### ISSUE TITLE
Tinymce editor Menu not in sync with menu button

### ISSUE THREAD

**Issue by Abhinavmwoe** [angular-ui/ui-tinymce#271]
my page is a bit long page as i am putting editor in my page quite below so wat happens is that when i click menu menubar opens on top of the page. i guess its a tinymce js issue..please guide me how to sync it with menubar.mayb putting it inside an iframe can fix it..if it can then how..please guide me..see screenshot.
![untitled](https://cloud.githubusercontent.com/assets/20181343/16406574/bc03583c-3d2c-11e6-9548-856046319c1c.png)

**Comment by deeg** [comment]
@Abhinavmwoe, please post a plunker reproducing the issue so we can look into it.

My guess is this is a CSS issue, and not necessarily a issue with this library, but rather maybe TinyMCE itself, or some of your CSS rules in your own application.

**Comment by Abhinavmwoe** [comment]
@deeg , yes i have solved this as it was a css issue but there is still a problem . once i load tinymce into a slider page on any page same thing occurs
